front-end/src/scenes/logIn/Components/NormalLoginForm.css

```diff
@@ -0,0 +1,25 @@
+.login-form{
+    display: block;
+    text-align: center;
+    align-self: center;
+}
+.login-form a {
+    color: #04233d;
+}
+.form-item {
+    margin-bottom: 10px;
+}
+.login-form-button {
+    background-color: #04233d;
+    border-color: #04233d;
+    width: 100%;
+}
+.logIn {
+     text-align: left;
+ }
+.check-box{
```

Comment:
Minor: Stylistic mistake, one more angry phrase about CSS modules, I don't want even hear about that while we on practice
Use global stylistic rules, here:
"check-box" - ok maybe some styles for styling checkbox
in reality definition - pulling left and (get out of a drawing flow) (in bootstrap such class called .pull-left)